documentation/asciidoc/computers/configuration/boot_folder.adoc

```diff
@@ -75,6 +75,8 @@ The boot folder will contain various xref:linux_kernel.adoc#kernel[kernel] image
 | Beta xref:config_txt.adoc#boot-options[64 bit kernel]. Raspberry Pi 2 with BCM2836 does not support 64-bit.
 |===
 
+IMPORTANT: On Raspberry Pi, the architecture reported by `lscpu` is the architecture of the running operating system, not the hardware.
```

Comment:
The "On Raspberry Pi" makes it sound like `lscpu` works differently on the Raspberry Pi than it does on other systems? Which I suspect probably isn't the case...

Perhaps it'd be better to just change the NOTE below to:
```
NOTE: The architecture reported by `lscpu` is `armv7l` for systems running a 32-bit kernel (i.e. everything except kernel8.img), and `aarch64` for systems running a 64-bit kernel. The `l` in the `armv7l` case refers to the architecture being little-endian, not `LPAE` as is indicated by the `l` in the `kernel7l.img` filename.
```
:man_shrugging:

Reply:
Sorry - you are correct. I must have been thinking about something different. You are correct that the `architecture` line output by `lscpu` does report the architecture of the running kernel on all platforms. This will add the required clarification, since the rest of the `lscpu` output does report on the hardware.